packages/server/src/db/index.js

```diff
@@ -147,7 +147,6 @@ async function getUser(id) {
             'tenants.id as tenant_id',
             'tenants.display_name as tenant_display_name',
             'tenants.main_agency_id as tenant_main_agency_id',
-            'tenants.uses_spoc_process as tenant_uses_spoc_process',
```

Comment:
I searched the entire project for the string "tenant_uses_spoc_process" and found no instances.